src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -580,16 +580,23 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
             }
         }
 
-        private static void ReportParameterErrors(
-            Symbol owner,
+
+        public static void ReportParameterErrors(
+            Symbol? owner,
             BaseParameterSyntax parameterSyntax,
-            ParameterSymbol parameter,
+            int ordinal,
+            bool isParams,
+            TypeWithAnnotations paramTypeWithAnnotations,
```

Comment:
>param [](http://example.com/codeflow?start=32&length=5)

Consider spelling out "parameter" for each of these parameters, or better yet, dropping the "param" prefix if the meaning is unambiguous.